wilds/datasets/ogbmolpcba_dataset.py

```diff
@@ -51,7 +51,7 @@ class OGBPCBADataset(WILDSDataset):
         https://github.com/snap-stanford/ogb/blob/master/LICENSE
     """
 
-    _dataset_name = 'ogbg-molpcba'
+    _dataset_name = 'ogb-molpcba'
```

Comment:
@kohpangwei @ssagawa Just wanted to double check if this change is okay.

Reply:
Yes, thanks!